tests/o11y/o11y.go

```diff
@@ -31,32 +28,63 @@ var _ = framework.O11ySuiteDescribe("O11Y E2E tests", Label("o11y", "HACBS"), Pe
 		})
 		AfterAll(func() {
 			if !CurrentSpecReport().Failed() {
-				// err := f.AsKubeAdmin.CommonController.DeleteAllPipelineRunsAndTasks(testNamespace)
-				// Expect(err).NotTo(HaveOccurred())
+				err := f.AsKubeAdmin.CommonController.DeleteAllPipelineRunsAndTasks(testNamespace)
+				Expect(err).NotTo(HaveOccurred())
 				Expect(f.SandboxController.DeleteUserSignup(f.UserName)).NotTo(BeFalse())
 			}
 		})
 
-		It("E2E sample test for upload 50MB pod", Pending, func() {
-			podNameRegex := ".*-upload-50mb-pod"
-			query := fmt.Sprintf("last_over_time(container_network_transmit_bytes_total{namespace='%s', pod=~'%s'}[1h])", testNamespace, podNameRegex)
+		It("E2E test to measure Egress pod by pushing images to quay", func() {
+
+			// Get Quay Organization from ENV
+			quayOrg := utils.GetEnv("DEFAULT_QUAY_ORG", "")
+			Expect(quayOrg).ToNot(BeEmpty())
+
+			// Get Quay Token from ENV
+			quayToken := utils.GetEnv("QUAY_TOKEN", "")
+			Expect(quayToken).ToNot(BeEmpty())
+
+			_, err := f.AsKubeAdmin.CommonController.CreateRegistryAuthSecret(o11yUserSecret, testNamespace, quayToken)
+			Expect(err).ToNot(HaveOccurred())
 
+			err = f.AsKubeAdmin.CommonController.LinkSecretToServiceAccount(testNamespace, o11yUserSecret, O11ySA)
+			Expect(err).ToNot(HaveOccurred())
+
+			pipelineRun, err := f.AsKubeAdmin.O11yController.QuayImagePushPipelineRun(quayOrg, o11yUserSecret, testNamespace)
+			Expect(err).NotTo(HaveOccurred())
+
+			// Wait for the pipeline run to succeed
+			Eventually(func() bool {
+				pr, err := f.AsKubeAdmin.TektonController.GetPipelineRun(pipelineRun.Name, pipelineRun.Namespace)
+				if err != nil {
+					GinkgoWriter.Println("PipelineRun has not been created yet \n", err, "PR Runs:", pr)
+					return false
+				}
+				if len(pr.Status.Conditions) > 0 {
+					for _, c := range pr.Status.Conditions {
+						if c.Type == "Succeeded" && c.Status == "True" {
+							return true
+						}
+					}
+				}
+				return false
+			}, pipelinerun_timeout, pipelinerun_interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to start or complete")
+
+			podNameRegex := ".*-buildah-quay-pod"
+			query := fmt.Sprintf("last_over_time(container_network_transmit_bytes_total{namespace='%s', pod=~'%s'}[1h])", testNamespace, podNameRegex)
 			result, err := f.AsKubeAdmin.O11yController.GetMetrics(query)
 			Expect(err).NotTo(HaveOccurred())
 
 			podNamesWithSize, err := f.AsKubeAdmin.O11yController.GetRegexPodNameWithSize(podNameRegex, result)
 			Expect(err).NotTo(HaveOccurred())
 
 			for podName, podSize := range podNamesWithSize {
-				GinkgoWriter.Printf("Pod: %s, Size: %.2f MB", podName, podSize)
 				// Range limits are measured as part of STONEO11Y-15
 				Expect(podSize).To(And(
-					BeNumerically(">=", 53.00),
-					BeNumerically("<=", 53.59),
-				), fmt.Sprintf("%s: %.2f MB is not within the expected range.", podName, podSize))
+					BeNumerically(">=", 106.00),
+					BeNumerically("<=", 108.50),
```

Comment:
What is this range based on? How confident are we that we're not going to introduce flakeyness here?